code/game/atoms.dm

```diff
@@ -607,3 +607,6 @@ its easier to just keep the beam vertical.
 		//An item is delivered on the cargo shuttle
 		//An item is purchased or dispensed from a vendor (Those things contain premade items and just release them)
 
+/atom/proc/get_cell()
+	return
```

Comment:
That proc only used in battery_module:
`/obj/item/weapon/computer_hardware/battery_module/get_cell()`
Why not defined it there?

Reply:
Ill revisit it later. This is proc that will be overriden in all items that can use cell to return it.

Reply:
> This is proc that will be overriden in all items that can use cell to return it.

That's not correct way for it.

Reply:
This PR includes getIDcard() that does the same but for id. How is that not correct way ?